### PR TITLE
Add constants for new DLRM hyperparameters

### PR DESCRIPTION
**Pull request by tgrel**

### CODE REVIEW

File: mlperf_logging/mllog/constants.py [hunk 1]
```diff
@@ -95,6 +95,8 @@
 OPT_LR_ALT_WARMUP_FUNC = "opt_learning_rate_alt_warmup_func"
 OPT_LR_DECAY_BOUNDARY_EPOCHS = "opt_learning_rate_decay_boundary_epochs"
 OPT_LR_DECAY_BOUNDARY_STEPS = "opt_learning_rate_decay_boundary_steps"
+OPT_LR_DECAY_START_STEP = "opt_learning_rate_decay_start_step"
+OPT_LR_DECAY_NUM_STEPS = "opt_learning_rate_decay_steps"
```
Comment (xyhuang) [inline mlperf_logging/mllog/constants.py:99]: this is already there in `OPT_LR_DECAY_STEPS`

File: mlperf_logging/mllog/constants.py [hunk 2]
```diff
@@ -95,6 +95,8 @@
 OPT_LR_ALT_WARMUP_FUNC = "opt_learning_rate_alt_warmup_func"
 OPT_LR_DECAY_BOUNDARY_EPOCHS = "opt_learning_rate_decay_boundary_epochs"
 OPT_LR_DECAY_BOUNDARY_STEPS = "opt_learning_rate_decay_boundary_steps"
+OPT_LR_DECAY_START_STEP = "opt_learning_rate_decay_start_step"
```
Comment (xyhuang) [inline mlperf_logging/mllog/constants.py:98]: could you please add it in alphabetical order, thx

File: mlperf_logging/mllog/constants.py (hunk 1 above)
Comment (tgrel) [reply mlperf_logging/mllog/constants.py]: sure, done

File: mlperf_logging/mllog/constants.py (hunk 2 above)
Comment (tgrel) [reply mlperf_logging/mllog/constants.py]: fixed

**Pull request merged**